aiida/restapi/translator/node.py

```diff
@@ -268,7 +269,7 @@ def _get_content(self):
             return {}
 
         # otherwise ...
-        node = self.qbobj.first()[0]
+        node = self.qbobj.all()[0][1]
```

Comment:
We should see with Leonid why this is not possible anymore.
If we don't find a solution, a ticket should be opened.

Reply:
The issue describing this #2808 I have fixed in PR #2824 so this change can now be reverted